src/dist/block_service/fds/fds_service.cpp

```diff
@@ -484,82 +498,139 @@ fds_file_object::fds_file_object(fds_service *s,
 
 fds_file_object::~fds_file_object() {}
 
-dsn::error_code fds_file_object::get_content(uint64_t pos,
-                                             int64_t length,
-                                             /*out*/ std::ostream &os,
-                                             /*out*/ uint64_t &transfered_bytes)
+error_code fds_file_object::get_file_meta()
```

Comment:
It seems that this function is used to read remote FDS by FDS client to get file's metadata, and `put_conent` and `create_file` also has similar code, I suggest you to refactor them too. If you would like  to refactor it in further pull request, I suggest adding some TODO comments.

Reply:
Yes, there are some code here. I will refactor in later pull request. 
I will add some TODO comments.